testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -331,3 +331,38 @@ def test_center_wrong_shape(self):
         weights = np.ones((self.ag.n_atoms, 2))
 
         assert_raises(TypeError, self.ag.center, weights)
+
+class TestRepresentations(object):
+    def setUp(self):
+        self.u = mda.Universe(CONECT)
```

Comment:
These tests are fine, but you don't really need to use `CONECT` here, you could use `make_Universe` to make a Universe that doesn't require file IO and should be a lot faster.